Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gradle:6.6-jdk11 AS build
```

Comment:
You use Gradle 6.8 in the wrapper. Probably, we need to use the same version here